components/dashboard/src/feedback-form/FeedbackComponent.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useState } from "react";
+import starry from "../images/feedback/starry-emoji.svg";
+import happy from "../images/feedback/happy-emoji.svg";
+import meh from "../images/feedback/meh-emoji.svg";
+import crying from "../images/feedback/crying-emoji.svg";
+import { trackEvent } from "../Analytics";
+
+function FeedbackComponent(props: { onClose: () => void; onSubmit: () => void; isModal: boolean }) {
+    const [text, setText] = useState<string>("");
+    const [selectedEmoji, setSelectedEmoji] = useState<number | undefined>();
+
+    const height = props.isModal ? "300px" : "";
+
+    const onSubmit = () => {
+        if (selectedEmoji) {
+            const feedbackObj = {
+                score: selectedEmoji,
+                feedback: text,
+                href: window.location.href,
+                path: window.location.pathname,
+            };
+            trackEvent("feedback_submitted", feedbackObj);
+        }
+
+        props.onSubmit();
+    };
+
+    const handleClick = (emojiScore: number) => {
+        setSelectedEmoji(emojiScore);
+    };
+
+    const emojiGroup = (width: number) => {
+        const emojiList = [
+            { id: 4, name: "starry", src: starry },
+            { id: 3, name: "happy", src: happy },
+            { id: 2, name: "meh", src: meh },
+            { id: 1, name: "crying", src: crying },
+        ];
+        return emojiList.map((emoji) => (
+            <button
+                className={
+                    "hover:scale-150 transform bg-transparent bottom-5 right-5 cursor-pointer " +
+                    (selectedEmoji === emoji.id ? "" : "grayed")
+                }
+                onClick={() => handleClick(emoji.id)}
+            >
+                <img src={emoji.src} alt={`${emoji.name} emoji`} width={width || "24px"} title={emoji.name} />
+            </button>
+        ));
+    };
+    return (
+        <>
+            <h3 className="mb-4">Send Feedback</h3>
+            {selectedEmoji ? (
+                <>
+                    <div className="flex flex-col -mx-6 px-6 py-4 border-t border-b border-gray-200 dark:border-gray-800 bg-white dark:bg-gray-900">
+                        <div className="relative">
+                            <div className="absolute flex bottom-5 right-5 -space-x-3">{emojiGroup(24)}</div>
+                            <textarea
+                                style={{ height: "160px", borderRadius: "6px" }}
+                                autoFocus
+                                className="w-full resize-none text-gray-400 dark:text-gray-400 focus:ring-0 focus:border-gray-400 dark:focus:border-gray-400 rounded-md border dark:bg-gray-900 dark:border-gray-500 bg-gray-800 border-gray-500"
```

Comment:
**issue**: Minor color fix, @laushinka sorry if all these variables sounds confusing! Once these components mature enough, we can extract all of these into separate reusable components. 🙏 

```suggestion
                                className="w-full resize-none text-gray-400 dark:text-gray-400 focus:ring-0 focus:border-gray-400 dark:focus:border-gray-400 rounded-md border dark:bg-gray-800 dark:border-gray-500 border-gray-500"
```

| BEFORE | AFTER | PARALLEL FIX |
|-|-|-|
| <img width="442" alt="modal-before" src="https://user-images.githubusercontent.com/120486/169306063-79776c86-ae22-4f33-a8ac-f44f2b83067b.png"> | <img width="442" alt="modal-after" src="https://user-images.githubusercontent.com/120486/169306073-e3e81b08-46d8-4847-bf2b-e30c4b194e7e.png"> | <img width="442" alt="modal-fix" src="https://user-images.githubusercontent.com/120486/169306083-171422fb-ad51-4fcf-a47f-259ce72fa0a5.png"> |